Assets/Students/HenryLiao/LaunchPad.cs

```diff
@@ -1,16 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class LaunchPad : MonoBehaviour
-{
-    public float launchSpeed = 10.0f;
-    private void OnTriggerEnter(Collider other)
-    {
-        Rigidbody ball = other.GetComponentInParent<Rigidbody>();
```

Comment:
Get the Ball class from the collider.
Check if ball is null.
Get rigidbody from ball and set velocity.